tests/guarded_transmute_many_pedantic/mod.rs

```diff
@@ -1,17 +1,19 @@
-use safe_transmute::{ErrorReason, GuardError, Error, guarded_transmute_many_pedantic};
+use safe_transmute::{ErrorReason, GuardError, Error, PedanticGuard};
+use safe_transmute::base::guarded_transmute_many;
 use self::super::LeToNative;
 
+type G = PedanticGuard;
 
 #[test]
 fn too_short() {
     unsafe {
-        assert_eq!(guarded_transmute_many_pedantic::<u16>(&[]),
+        assert_eq!(guarded_transmute_many::<u16, G>(&[]),
```

Comment:
Module name doesn't match tested funxion – are we letting this slide 'till the test reorganisation?

Reply:
👌